src/main/java/org/craftercms/studio/impl/v2/service/security/UserServiceImpl.java

```diff
@@ -147,7 +147,10 @@ public User createUser(User user) throws UserAlreadyExistsException, ServiceLaye
 
     @Override
     @HasPermission(type = DefaultPermission.class, action = PERMISSION_UPDATE_USERS)
-    public void updateUser(User user) throws ServiceLayerException, UserNotFoundException, AuthenticationException {
+    public void updateUser(User user) throws ServiceLayerException, UserNotFoundException, AuthenticationException, UserExternallyManagedException {
+        checkExternallyManagedUsers(new ArrayList<>(Arrays.asList(user.getId())),
```

Comment:
Can we simplify this to something like:
```
checkExternallyManagedUsers(Arrays.asList(user.getId()),
                Arrays.asList(user.getUsername()));
```
?

Reply:
Yes, fixed this. Thank you!